AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginIntegrationTests/PrimaryKey/schema.graphql

```diff
@@ -0,0 +1,31 @@
+# This "input" configures a global authorization rule to enable public access to
+# all models in this schema. Learn more about authorization rules here: https://docs.amplify.aws/cli/graphql/authorization-rules
```

Comment:
i think this fiile should live with the AmplifyTestCommon like the others and have it be reference in the README.md in DataStore's integration test target. in the README.md

Reply:
yup, deleted and updated the schema in `AmplifyTestCommon` in https://github.com/aws-amplify/amplify-ios/pull/1878/files